tests/unit/utils/formatUserDetails.test.ts

```diff
@@ -22,4 +25,17 @@ describe("formatUserDetails function", () => {
     const expectedFormattedDetails = `## User Details\n${userFullName}\n${discordJoinedAt}\n${userState}`;
     expect(formattedDetails).toEqual(expectedFormattedDetails);
   });
+
+  it("should return empty string if discordJoinedAt is undefined", () => {
+    const formattedDetails = formatUserDetails(
+      userWithoutDiscordJoinedAtResponse
+    ).trim();
+    const userFullName = `**Full Name :** Jyotsna Mehta`;
+    const discordJoinedAt = `**RDS Discord Joined at :** ${convertTimeStamp(
+      userWithoutDiscordJoinedAtResponse
+    )}`;
+    const userState = `**State :** IDLE`;
+    const expectedFormattedDetails = `## User Details\n${userFullName}\n${discordJoinedAt}\n${userState}`;
+    expect(formattedDetails).toEqual(expectedFormattedDetails);
+  });
```

Comment:
Can you also give an example of how it may look like and when this case is going to occur?

Reply:
- Hey previously I was providing an empty string which means that if the value for discord joined at is not present the field will be empty. But there should be a meaningful phrase or a message provided So I will do the same.
## Before : 
![image](https://github.com/Real-Dev-Squad/discord-slash-commands/assets/53934353/d0ee56ea-69fe-4bec-9fd1-4f7369239941)

## After : 
// To be decided

- About the second half of your question, this case will most likely never occur in my opinion but the reason I am handling this edge case is because of two reasons : 
1. Typescript error : 
- **userDetails.user?.discordJoinedAt** may be undefined. In TypeScript, when you try to pass an undefined value as an argument to the Date constructor, you get a type error because the Date constructor expects a valid argument, which should be a string or number.

- To fix this error, I have add a check to ensure that timestamp is not undefined before creating a Date object
## Before : 
![image](https://github.com/Real-Dev-Squad/discord-slash-commands/assets/53934353/6e999a8f-15e0-43b6-8a61-126f733e5ab1)

## After : 
![image](https://github.com/Real-Dev-Squad/discord-slash-commands/assets/53934353/2673b4be-2a20-41d1-bf89-0c227465947f)

Reply:
I was asking about when did this case arrive in which user exist in db and have no discord joined at